script.js

```diff
@@ -1,4 +1,47 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+var main = function (userInputChoice) {
```

Comment:
Overall i think you got the idea of what to do and how to do it but code is a little messy. make a habit to use comments to describe what is going on or what you are trying to achieve. try to run in your head (or use pseudo code) the flow of how you want the script to play out